python/ola/PidStore.py

```diff
@@ -65,7 +69,7 @@ class UnpackException(Error):
 
 
 class MissingPLASAPIDs(Error):
```

Comment:
I wonder if we should really have renamed this (or kept the old one for backwards compatibility and subclass or re-raise or something)?

Reply:
The name hasn't changed since it was introduced, AFAICT.  The comment change was one of the changes brought in from master.

Reply:
Ah okay, one for after this mammoth PR is in then!